subprojects/kotlin-dsl-plugins/kotlin-dsl-plugins.gradle.kts

```diff
@@ -35,10 +34,9 @@ gradlebuildJava {
     moduleType = ModuleType.INTERNAL
 }
 
-withCompileOnlyGradleApiModulesWithParameterNames(":pluginDevelopment")
-
 dependencies {
     compileOnly(project(":kotlinDsl"))
+    compileOnly(project(":pluginDevelopment"))
```

Comment:
:question: those could now be `compile` instead, correct?

Reply:
We don't want those dependencies to be visible in the plugin dependency graph, do we?

Reply:
Right, in `:kotlinDslPlugins` we don't. We can leave it like that in other projects too.